### ISSUE TITLE
Visdom Image Overlapping

### ISSUE THREAD

**Issue by mightyabra** [pytorch/tnt#121]
I am working on a neural network code and i am using Visdom server for my visualization. Everything was working well until last two days, my confusion matrix started replicating itself and covering my other graphs (test accuracy graph, test loss graph). My training accuracy graph is now totally messed up. I now have three confusion matrix instead of one.

Software used: Anaconda (using Pytorch)
OS: Windows 10
Browser: Chrome

1. I have uninstalled and reinstalled Anaconda

2. I've uninstalled and reinstalled visdom.

3. i seem not to know what i am doing wrong.

4. I am using the code on a different machine and it is working well (as in all the graphs are showing correctly and in their place).

![Visdom Error](https://user-images.githubusercontent.com/40210809/64076441-8ccdfc00-ccb4-11e9-81e1-6c49c52fd552.PNG)
![code](https://user-images.githubusercontent.com/40210809/64076442-8e97bf80-ccb4-11e9-80f4-de1e7d4b9c34.PNG)

**Comment by szagoruyko** [comment]
odd, I never looked at this functionality, no ideas :/

**Comment by mightyabra** [comment]
Thanks guys for your suggestions.
